packages/core/src/autoscaling/snapshotter.ts

```diff
@@ -195,13 +195,8 @@ export class Snapshotter {
         // Start snapshotting.
         this.eventLoopInterval = betterSetInterval(this._snapshotEventLoop.bind(this), this.eventLoopSnapshotIntervalMillis);
         this.clientInterval = betterSetInterval(this._snapshotClient.bind(this), this.clientSnapshotIntervalMillis);
-        if (isAtHome()) {
-            events.on(ACTOR_EVENT_NAMES.SYSTEM_INFO, this._snapshotCpuOnPlatform);
-            events.on(ACTOR_EVENT_NAMES.SYSTEM_INFO, this._snapshotMemoryOnPlatform);
-        } else {
-            this.cpuInterval = betterSetInterval(this._snapshotCpuOnLocal.bind(this), this.cpuSnapshotIntervalMillis);
-            this.memoryInterval = betterSetInterval(this._snapshotMemoryOnLocal.bind(this), this.memorySnapshotIntervalMillis);
-        }
```

Comment:
let's not change this, we need to abstract the platform events, not just ignore them :]

Reply:
Will bring them back but commented out like before, is that alright?

Reply:
Sure, just add a `FIXME` comment there (I do prefer those for things that really should not get published, while `TODO` is something we can improve on later).